<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ezs013" ignored="0" name="Test" tests="9" failures="8" timestamp="20 Mar 2018 14:57:38 GMT" time="22.031" errors="0">
  <testcase name="UniversalSettings" time="14.072" classname="com.eDocs.solidCalculation.Default_Limit"/>
  <testcase name="P1_Active1_patch" time="5.981" classname="com.eDocs.patchCalculation.PatchCalculation">
    <failure type="java.util.NoSuchElementException">
      <![CDATA[java.util.NoSuchElementException
	at java.util.ArrayList$Itr.next(ArrayList.java:839)
	at java.util.Collections.min(Collections.java:656)
	at com.eDocs.patchCalculation.PatchCalculation.P1_Active1_patch(PatchCalculation.java:476)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- P1_Active1_patch -->
  <testcase name="P1_Active2_patch" time="0.418" classname="com.eDocs.patchCalculation.PatchCalculation">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 1, Size: 1">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
	at java.util.ArrayList.rangeCheck(ArrayList.java:638)
	at java.util.ArrayList.get(ArrayList.java:414)
	at com.eDocs.patchCalculation.L0forpatch.calculate_P1_active2_L0(L0forpatch.java:152)
	at com.eDocs.patchCalculation.PatchCalculation.P1_Active2_patch(PatchCalculation.java:586)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- P1_Active2_patch -->
  <testcase name="P2_Active1_patch" time="0.269" classname="com.eDocs.patchCalculation.PatchCalculation">
    <failure type="com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException" message="Unknown column &amp;apos;min_no_ofpatches_worn_at_one_time&amp;apos; in &amp;apos;field list&amp;apos;">
      <![CDATA[com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'min_no_ofpatches_worn_at_one_time' in 'field list'
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411)
	at com.mysql.jdbc.Util.getInstance(Util.java:386)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1052)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3609)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3541)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2002)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2163)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2618)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2568)
	at com.mysql.jdbc.StatementImpl.executeQuery(StatementImpl.java:1557)
	at com.eDocs.patchCalculation.L0forpatch.calculate_P2_active1_L0(L0forpatch.java:250)
	at com.eDocs.patchCalculation.PatchCalculation.P2_Active1_patch(PatchCalculation.java:959)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- P2_Active1_patch -->
  <testcase name="P2_Active2_patch" time="0.269" classname="com.eDocs.patchCalculation.PatchCalculation">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 1, Size: 1">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
	at java.util.ArrayList.rangeCheck(ArrayList.java:638)
	at java.util.ArrayList.get(ArrayList.java:414)
	at com.eDocs.patchCalculation.L0forpatch.calculate_P2_active2_L0(L0forpatch.java:347)
	at com.eDocs.patchCalculation.PatchCalculation.P2_Active2_patch(PatchCalculation.java:1327)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- P2_Active2_patch -->
  <testcase name="P3_Active1_patch" time="0.292" classname="com.eDocs.patchCalculation.PatchCalculation">
    <failure type="com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException" message="Unknown column &amp;apos;min_no_ofpatches_worn_at_one_time&amp;apos; in &amp;apos;field list&amp;apos;">
      <![CDATA[com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'min_no_ofpatches_worn_at_one_time' in 'field list'
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411)
	at com.mysql.jdbc.Util.getInstance(Util.java:386)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1052)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3609)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3541)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2002)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2163)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2618)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2568)
	at com.mysql.jdbc.StatementImpl.executeQuery(StatementImpl.java:1557)
	at com.eDocs.patchCalculation.L0forpatch.calculate_P3_active1_L0(L0forpatch.java:446)
	at com.eDocs.patchCalculation.PatchCalculation.P3_Active1_patch(PatchCalculation.java:1695)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- P3_Active1_patch -->
  <testcase name="P3_Active2_patch" time="0.255" classname="com.eDocs.patchCalculation.PatchCalculation">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 1, Size: 1">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
	at java.util.ArrayList.rangeCheck(ArrayList.java:638)
	at java.util.ArrayList.get(ArrayList.java:414)
	at com.eDocs.patchCalculation.L0forpatch.calculate_P3_active2_L0(L0forpatch.java:545)
	at com.eDocs.patchCalculation.PatchCalculation.P3_Active2_patch(PatchCalculation.java:2062)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- P3_Active2_patch -->
  <testcase name="P4_Active1_patch" time="0.199" classname="com.eDocs.patchCalculation.PatchCalculation">
    <failure type="com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException" message="Unknown column &amp;apos;min_no_ofpatches_worn_at_one_time&amp;apos; in &amp;apos;field list&amp;apos;">
      <![CDATA[com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'min_no_ofpatches_worn_at_one_time' in 'field list'
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411)
	at com.mysql.jdbc.Util.getInstance(Util.java:386)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1052)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3609)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3541)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2002)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2163)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2618)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2568)
	at com.mysql.jdbc.StatementImpl.executeQuery(StatementImpl.java:1557)
	at com.eDocs.patchCalculation.L0forpatch.calculate_P4_active1_L0(L0forpatch.java:644)
	at com.eDocs.patchCalculation.PatchCalculation.P4_Active1_patch(PatchCalculation.java:2433)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- P4_Active1_patch -->
  <testcase name="P4_Active2_patch" time="0.22" classname="com.eDocs.patchCalculation.PatchCalculation">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 1, Size: 1">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
	at java.util.ArrayList.rangeCheck(ArrayList.java:638)
	at java.util.ArrayList.get(ArrayList.java:414)
	at com.eDocs.patchCalculation.L0forpatch.calculate_P4_active2_L0(L0forpatch.java:742)
	at com.eDocs.patchCalculation.PatchCalculation.P4_Active2_patch(PatchCalculation.java:2800)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- P4_Active2_patch -->
</testsuite> <!-- Test -->
